bigtable-hbase-parent/bigtable-hbase/src/main/java/com/google/cloud/bigtable/hbase/BatchExecutor.java

```diff
@@ -340,8 +317,10 @@ public void batch(List<? extends Row> actions, @Nullable Object[] results)
    */
   public Result[] batch(List<? extends Row> actions) throws IOException {
     try {
-      Result[] results = new Result[actions.size()];
-      batch(actions, results);
+      Object[] resultsOrErrors = new Object[actions.size()];
+      batch(actions, resultsOrErrors);
+      Result[] results = new Result[resultsOrErrors.length];
+      System.arraycopy(resultsOrErrors, 0, results, 0, results.length);
```

Comment:
resultsOrErrors can't contain exceptions at this point. batch would've thrown an exception if any action failed